guide/creating-your-bot/commands-with-user-input.md

```diff
@@ -72,7 +72,7 @@ else if (command === 'args-info') {
 }
 ```
 
-So if the first argument provided is equal to "foo", then send back "bar". Otherwise, just send back the argument the user provided.
+So if the first argument provided is equal to "foo", then send back "bar". Otherwise, just send back the argument the user-provided.
```

Comment:
```suggestion
So if the first argument provided is equal to "foo", then send back "bar". Otherwise, just send back the argument the user provided.
```
Your auto-correct is garbage.